packages/cli/src/Generate.ts

```diff
@@ -290,33 +288,13 @@ import { PrismaClient } from '${importPath}/${isDeno ? 'deno/' : ''}edge${isDeno
 const prisma = new PrismaClient()`)}
 ${dim('```')}
 
-More information: https://pris.ly/d/client`
-          } else if (generator?.isCustomOutput && !isDeno) {
-            hint = `
-Prisma Client has been generated to a custom path:
-
-${bold('1. Make it a dependency of your project')}
-${dim('```')}
-${grey(`# adapt this relative path if needed`)}
-${bold(blue(await getPackageCmd(cwd, 'add', `db@${agentPathPrefix}${importPath}`)))}
-${dim('```')}
-
-More information: https://pris.ly/d/custom-output
-
-${bold('2. Start using Prisma Client')}
-${dim('```')}
-${highlightTS(`\
-import { PrismaClient } from 'db'
-const prisma = new PrismaClient()`)}
-${dim('```')}
-
 More information: https://pris.ly/d/client`
           } else {
             hint = `
 ${bold('Start using Prisma Client')}
 ${dim('```')}
 ${highlightTS(`\
-import { PrismaClient } from ${importPath}
+import { PrismaClient } from '${importPath}'
```

Comment:
Was this a bug before?

Reply:
It seems it was a typo in our stdout message, yes